e2e/scripts/dts-import.js

```diff
@@ -0,0 +1,81 @@
+import { resolve } from 'path';
+import { ALLOWED_CONTENT_TYPES, CUSTOM_TRANSFER_TOKEN_ACCESS_KEY } from '../constants';
+
+const {
+  file: {
+    providers: { createLocalFileSourceProvider },
+  },
+  strapi: {
+    providers: { createRemoteStrapiDestinationProvider },
+  },
+  engine: { createTransferEngine },
+} = require('@strapi/data-transfer');
+
+/**
+ * Reset the DB and import data from a DTS backup
+ * This function ensures we keep all admin user's and roles in the DB
```

Comment:
Why do we want that? This way we can't test/reset things that affect admin users or roles.

We should take advantage of the ability to reset the entire Strapi to a known state.

Reply:
> This way we can't test/reset things that affect admin users or roles

Why do you think that?

The issue we're going to have is needing to setup the super-admin every test, I think it's impractical and if we want to test pieces unique to users that require them being wiped from the DB then I would suggest we actually do a workaround for that unique test suite?